packages/terra-application-layout/src/menu/_ApplicationMenu.jsx

```diff
@@ -0,0 +1,179 @@
+import React from 'react';
```

Comment:
Should this be its own component in terra-framework that the app consumes?

Reply:
The functionality it provides is so closely tied to that of the ApplicationLayout that I don't see a lot of value releasing it separately. I think it'd be more likely that the `terra-application-header-layout`, which is already in its own package, would be consumed instead.